Tests/Packet++Test/main.cpp

```diff
@@ -7113,6 +7113,50 @@ PACKETPP_TEST(RadiusLayerEditTest)
 	PACKETPP_TEST_PASSED;
 }
 
+PACKETPP_TEST(MacAddressTests)

```

Comment:
There is already existing UT for `MacAddress` in Pcap++Test: https://github.com/seladb/PcapPlusPlus/blob/7bc25adc8f5e2c0dc59710a3fa94b6956498a298/Tests/Pcap%2B%2BTest/main.cpp#L663

Can you move these UTs to there?

Reply:
Ok

Reply:
Done